### PR TITLE
Use renamed method planParamsToQueryAsync

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR matches changes from https://github.com/opentripplanner/otp-ui/pull/185.

When unblocking, update the package dependencies!!

To test, follow the same steps as in #182.

### CODE REVIEW

**Comment by binh-dam-ibigroup** [comment]
Blocked pending release of opentripplanner/otp-ui#185.

File: yarn.lock [hunk 1]
```diff
@@ -1368,6 +1368,20 @@
     prop-types "^15.7.2"
     qs "^6.9.1"
 
+"@opentripplanner/core-utils@^1.2.1":
```
Comment (evansiroky) [inline yarn.lock:1371]: Please use [this tool](https://www.npmjs.com/package/yarn-deduplicate) to fix deduplicate this entry in yarn.lock.

**Pull request merged**